splinterd/src/daemon.rs

```diff
@@ -857,9 +856,10 @@ fn build_biome_routes(
     store_factory: &dyn splinter::store::StoreFactory,
 ) -> Result<BiomeRestResourceManager, StartError> {
     info!("Adding biome routes");
+    // Allowing unused_mut because `biome_rest_provider_builder` must be mutable if features
+    // biome-credentials or biome-key-management are enabled
+    #[allow(unused_mut)]
```

Comment:
Should we only create this if at least one of "biome-credentials" or "biome-key-management" is enabled?

Reply:
Good idea; updated.